scripts/grest-helper-scripts/grest-poll.sh

```diff
@@ -72,6 +72,14 @@ function usage() {
   exit 1
 }
 
+function is_up() {
```

Comment:
Function name mismatch:
```suggestion
function chk_is_up() {
```